src/models/quote.js

```diff
@@ -1,38 +1,29 @@
 'use strict'
 
-const request = require('co-request')
 const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
 const UnacceptableAmountError = require('../errors/unacceptable-amount-error')
 const AssetsNotTradedError = require('../errors/assets-not-traded-error')
-const ExternalError = require('../errors/external-error')
 const DEFAULT_DESTINATION_EXPIRY = 5 // seconds
 
 function * makeQuoteQuery (params) {
-  const sourceLedger = params.source_ledger ||
-    (yield getAccountLedger(params.source_account))
-  const destinationLedger = params.destination_ledger ||
-    (yield getAccountLedger(params.destination_account))
+  const sourceLedger = addressToLedger(params.source_address)
+  const destinationLedger = addressToLedger(params.destination_address)
   const explain = params.explain === 'true'
   return {
     sourceLedger,
     destinationLedger,
     sourceAmount: params.source_amount,
     destinationAmount: params.destination_amount,
+    destinationPrecisionAndScale: params.destination_precision && {
+      precision: params.destination_precision,
+      scale: params.destination_scale
+    },
     explain
   }
 }
 
-function * getAccountLedger (account) {
-  const res = yield request({
-    method: 'get',
-    uri: account,
-    json: true
-  })
-  const ledger = res.body && res.body.ledger
-  if (res.statusCode !== 200 || !ledger) {
-    throw new ExternalError('Unable to identify ledger from account: ' + account)
-  }
-  return ledger
+function addressToLedger (address) {
+  return address.split('.').slice(0, -1).join('.')
```

Comment:
The use case is I'm `us.fed.wf.stefan`. I want to make a subledger, but I don't want to worry about getting permission, peering with connectors, broadcasting routes etc. So I just call my subledger `us.fed.wf.stefan.` and assign addresses like `us.fed.wf.stefan.phone`. Every payment to `us.fed.wf.stefan.phone` is going to at some point show up in my account `us.fed.wf.stefan` unless somebody has a more direct route to my subledger. So I can run a connector against my account and route payments to my subledger successfully.

Subledgering is useful and extremely common. Banks run subledgers to the Fed, wallets usually run a subledger of whatever their bank is, etc. So it makes sense to have a format that makes subledgering natural and easy.

We could make a convention that we leave a trailing dot when we're talking about a ledger, i.e. `us.fed.wf.stefan` is referring to my account and `us.fed.wf.stefan.` is talking about my subledger.